client/js/index.js

```diff
@@ -154,6 +154,7 @@ function dataHandler(inputData) {
     inputData.gps_lat,
     inputData.gps_long,
     inputData.gps_alt,
+    4,
```

Comment:
For the record that's around 11m accuracy. Maybe document this for future skeptics like myself?